docs/modules/ROOT/pages/references/architecture/control-api-org.adoc

```diff
@@ -63,24 +63,82 @@ metadata:
 
 |===
 
-== Resource filter
+=== Resource filter
 
 The virtual resource is a filtered view of `Namespaces`.
 The filter uses the following heuristic:
 
 * API version: `v1`
 * Kind: `Namespace`
 * Label: `appuio.io/resource-type=organization`
-* Subject is bound to one of the defined `ClusterRole` resources.
 
-== RBAC and Cluster roles
+== Access Control
 
-These are `ClusterRole` resources which are bound to a subject by a namespaced `RoleBinding`:
+Access control for `Organizations` uses standard Kubernetes RBAC rules, but `Organizations` are represented by two different resources.
+
+. The `organizations.organizations.appuio.io` resource is the cluster-scoped resource we actually access.
```

Comment:
Who is `we` in this context? I think we should clearly state which actor is "accessing" them. we failed to previously document who is the main entity that manages or interacts with these resources.
```suggestion
. The `organizations.organizations.appuio.io` resource is the cluster-scoped resource any authenticated human user actually interacts with.
```